drivers/led_strip/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ zephyr_library_sources_ifdef(CONFIG_APA102_STRIP        apa102.c)
 zephyr_library_sources_ifdef(CONFIG_LPD880X_STRIP       lpd880x.c)
 zephyr_library_sources_ifdef(CONFIG_WS2812_STRIP_GPIO   ws2812_gpio.c)
```

Comment:
In the commit message, please use upper cases for UART and RGB. These are acronyms.